README.rst

```diff
@@ -45,6 +45,15 @@ them yourselves using your own Python scripts.
 Module and CLI Guide
 ~~~~~~~~~~~~~~~~~~~~
 
+If you have the project installed in a virtualenv, you can use the following console scripts:
+
+* ``parsely_redshift`` = ``parsely_raw_data.redshift``
+* ``parsely_bigquery`` = ``parsely_raw_data.bigquery``
+* ``parsely_s3`` = ``parsely_raw_data.s3``
+* ``parsely_stream`` = ``parsely_raw_data.stream``
+
+Alternately, you can run each module like this:
```

Comment:
Might want to add that that only works from the directory where this is cloned if you haven't installed it with `pip`. When installed, it should work from anywhere.